WikiWatershed/model-my-watershed
# Update Default N Efficiencies for Crops, Tillage

## Overview

Updates default Nitrogen efficiencies for Cover Crops and Conservation Tillage as specified. Also refactors the code to be less redundant.

Note that this is targeted against the `release/1.26.0` branch, not `develop`.

Connects #3223 

### Demo

![image](https://user-images.githubusercontent.com/1430060/71914947-b27a7e80-3148-11ea-9bd6-2b8b8fb6da1c.png)

## Testing Instructions

* Checkout this branch and reload `app` and `worker` (necessary because the changes in `gwlfe_settings` are not picked up until the server is restarted)
    ```console
    $ vagrant reload app worker
    ```
* Rebuild the bundle
    ```console
    $ ./scripts/bundle.sh --debug
    ```
* Go to [:8000](http://localhost:8000/) and select / draw a shape
* Model it using MapShed. Click "Add changes to this area"
* Click the "Settings" button in the top-right corner
* Ensure the dialog has the new values as shown
